src/lasp_core.erl

```diff
@@ -361,7 +361,7 @@ update(Id, Operation, Actor, Store) ->
     {ok, var()} | not_found().
 update(Id, Operation, Actor, MetadataFun, Store) ->
     {ok, #dv{value=Value0, type=Type}} = do(get, [Store, Id]),
-    {ok, Value} = lasp_type:update(Type, Operation, Actor, Value0),
+    {ok, Value} = lasp_type:update(Type, Operation, {Id, Actor}, Value0),
```

Comment:
Rather, a better question is *how* unique does this need to be?  Is this what's being used for `next()` in the semantics that you mailed over earlier?